### DISCUSSION TITLE
Take temperature from an analog dial with ESPHOME and hass

### DISCUSSION THREAD

**Discussion by gzebedin** [jomjol/AI-on-the-edge-device#2944] [General]
Hello,
ist it possible to get the (waterboiler) temperature from an analog dial like this one?
https://i.ebayimg.com/images/g/11IAAOSwj2RbB~-c/s-l1600.jpg

I want to get the actual temperature into my homeassistant.

Thank you

**Comment by caco3** [comment]
Please have a look on https://github.com/jomjol/AI-on-the-edge-device/discussions/1090
